0009-RHEL-19149-Pass-FUSE_PARALLEL_DIROPS-to-kernel.patch

```diff
@@ -0,0 +1,25 @@
+From c9905341ea34ff9acbc11b3c53ba8bcea35eeed8 Mon Sep 17 00:00:00 2001
+From: fdinoff <fdinoff@google.com>
+Date: Thu, 16 Nov 2023 06:23:20 -0500
+Subject: [PATCH] Pass FUSE_PARALLEL_DIROPS to kernel (#861)
+
+This tells the kernel that parallel lookup/readdir operations are
+supported. This is enabled by default but was not passed to the kernel
+so you always get the synchronized version.
+---
+ lib/fuse_lowlevel.c | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/lib/fuse_lowlevel.c b/lib/fuse_lowlevel.c
+index fdef193bd..ea985360d 100644
+--- a/lib/fuse_lowlevel.c
++++ b/lib/fuse_lowlevel.c
+@@ -2141,6 +2141,8 @@ void do_init(fuse_req_t req, fuse_ino_t nodeid, const void *inarg)
+ 		outargflags |= FUSE_ASYNC_DIO;
+ 	if (se->conn.want & FUSE_CAP_WRITEBACK_CACHE)
+ 		outargflags |= FUSE_WRITEBACK_CACHE;
++	if (se->conn.want & FUSE_CAP_PARALLEL_DIROPS)
```

Comment:
There was a follow on patch to the original to disable this by default because it breaks some fuse instances that are not prepared for it.  Did we get that patch too?  We would not want to break non-dfuse mounts.

Reply:
Indeed, I am aware of the follow-on patch.  The patch above is really just a placeholder until we get an official RH-blessed backport patch, which is apparently in progress now that the issue that resulted in the patch you refer to has been resolved between RH and the fuse maintainers.

I appreciate the heads-up though.